src/main/java/com/twitter/whiskey/util/ZlibInflater.java

```diff
@@ -171,13 +171,14 @@ public void reset() {
 
     @Override
     public void end() {
-        inflater.end();
+        if (inflater != null) inflater.end();
+        super.end();
```

Comment:
the Inflater.java from roboelectric has a closeGuard. This crashes the TwitterAndroid app when each instantiation does not have a corresponding end call before finalize. I agree that this class indeed is weird in that it both extends and has an inflater. But I didn't want to fix that issue in this PR.